pkg/gui/controllers/options_menu_action.go

```diff
@@ -18,23 +18,33 @@ func (self *OptionsMenuAction) Call() error {
 		return nil
 	}
 
-	bindings := self.getBindings(ctx)
+	local, global, navigation := self.getBindings(ctx)
 
-	menuItems := lo.Map(bindings, func(binding *types.Binding, _ int) *types.MenuItem {
-		return &types.MenuItem{
-			OpensMenu: binding.OpensMenu,
-			Label:     binding.Description,
-			OnPress: func() error {
-				if binding.Handler == nil {
-					return nil
+	menuItems := []*types.MenuItem{}
+
+	appendBindings := func(bindings []*types.Binding, section string) {
+		menuItems = append(menuItems,
+			lo.Map(bindings, func(binding *types.Binding, _ int) *types.MenuItem {
+				return &types.MenuItem{
+					OpensMenu: binding.OpensMenu,
+					Label:     binding.Description,
+					OnPress: func() error {
+						if binding.Handler == nil {
+							return nil
+						}
+
+						return binding.Handler()
+					},
+					Key:     binding.Key,
+					Tooltip: binding.Tooltip,
+					Section: section,
 				}
+			})...)
+	}
 
-				return binding.Handler()
-			},
-			Key:     binding.Key,
-			Tooltip: binding.Tooltip,
-		}
-	})
+	appendBindings(local, "Local")
+	appendBindings(global, "Global")
+	appendBindings(navigation, "Navigation")
```

Comment:
I wasn't sure if we want a separate section for Navigation; we could also add this to the Global section, as it is now.

Reply:
I like having a separate section. I'm thinking that in a separate PR we should go and do a better job breaking up the keybindings into more headings.